tutorials/shaders/compute_shaders.rst

```diff
@@ -189,18 +283,22 @@ The steps we need to do to compute our result are:
     rd.ComputeListDispatch(computeList, xGroups: 5, yGroups: 1, zGroups: 1);
     rd.ComputeListEnd();
 
-Note that we are dispatching the compute shader with 5 work groups in the x-axis, and one in the others.
-Since we have 2 local invocations in the x-axis (specified in our shader) 10 compute shader invocations will be launched in total.
-If you read or write to indices outside of the range of your buffer, you may access memory outside of your shaders control or parts of other variables which may cause issues on some hardware.
-
+Note that we are dispatching the compute shader with 5 work groups in the
+x-axis, and one in the others. Since we have 2 local invocations in the x-axis
+(specified in our shader) 10 compute shader invocations will be launched in
+total. If you read or write to indices outside of the range of your buffer, you
+may access memory outside of your shaders control or parts of other variables
+which may cause issues on some hardware.
 
 Execute a compute shader
 ------------------------
 
-After all of this we are done, kind of.
-We still need to execute our pipeline, everything we did so far was only definition not execution.
+After all of this we are almost done, we still need to execute our pipeline. So
+far we have only recorded what we would like the GPU to do, we have not actually
+run the shader program.
 
-To execute our compute shader we just need to submit the pipeline to the GPU and wait for the execution to finish:
+To execute our compute shader we just need to submit the pipeline to the GPU and
```

Comment:
Avoid "just":

```suggestion
To execute our compute shader, we need to submit the pipeline to the GPU and
```

Reply:
removed "just", but I don't think a comma belongs here, it fragments the sentence awkwardly